index.css

```
@@ -0,0 +1,186 @@
+body {
+    background-color: #f2f2f2;
+    color: #333;
+    font-weight: 400;
+    line-height: 1.3;
+    margin: 32px;
+    font-family: Arial, Arial, Helvetica, sans-serif;
```

Your font stack repeats `Arial` twice:
```suggestion
    font-family: Arial, Helvetica, sans-serif;
```